src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -234,4 +248,42 @@ void testIsPaperCaseAttending() {
 
         assertThat(result).isFalse();
     }
+
+
+    private static Stream<Arguments> buildHearingChannelRequests() {
```

Comment:
This is not used, please delete. 
As a matter of note, we try to avoid using @MethodSource in favour of @CsvSource, or better yet @EnumSource. The reason is clarity - when looking at a test with the latter sources, it is much simpler to understand what is going on. In the case of enum, it is more easily extensible if said enum is changed at a later date.

Reply:
Removed the unwanted the code.